go.mod

```diff
@@ -34,7 +34,7 @@ replace (
 )
 
 replace (
-	github.com/boson-project/faas => github.com/boson-project/faas v0.8.0
+	github.com/boson-project/faas => github.com/boson-project/faas v0.9.0
```

Comment:
Isn't it good enough to set the version on the original dependency ? I thinks this is the way as it should work and `replace` should be reserved only for "special cases".

Reply:
I think this was done for the control and ease of grep/replace.

Reply:
Yep, Navid described my thoughts quiet well here. :)

Reply:
RIght now the replace itself is created by our "tooling". In addition the revert was a good call to avoid duplicate `replace` blocks. I'll take a look how to improve this for the future release.